### ISSUE TITLE
script no longer works for me anyway?

### ISSUE THREAD

**Issue by stapuft** [thorio/KGrabber#16]
on both kimcartoon and kissanime, the "experimental grabber" feature is broken and must be disabled in order to get any episode links. with it enabled you constantly get this message,
"error: server not available or captcha"
after disabling this feature the script works again, but only in the old way, where it goes page by page very slowly.

**Comment by stapuft** [comment]
also rapidvideo doesnt work even with experimental grabber unchecked.

**Comment by thorio** [comment]
Works fine on Dr. Stone ep3 rapidvideo in Chrome and Firefox.
Please provide details on the show, episode, server, browser etc.

Also, kissanime doesn't have any experimental grabbers assigned.

**Comment by stapuft** [comment]
firefox 68.0 64 bit, windows 10
every show I try every episode I try, every server.

okay, thats odd the option is still there, and unchecking it makes you go the long way to getting the episode (actually loads the page then gets the episode link) whereas having it checked just instantly puts that new box up top and gives you the episode link?
for example with it checked I select rapidvideo for example and instantly this pops up without even refreshing the page
![image](https://user-images.githubusercontent.com/46431157/61625635-326b6580-ac49-11e9-8bdc-d2c5f803200e.png)
but with it unchecked it loads the videos page and then goes back to the main page but it happens so fast the video doesnt even load on the page before the page changes
![image](https://user-images.githubusercontent.com/46431157/61625807-97bf5680-ac49-11e9-8366-b7b2b6cf6683.png)
only to be greeted by this
![image](https://user-images.githubusercontent.com/46431157/61625832-a86fcc80-ac49-11e9-8de7-f39271f131d5.png)
but if you go to the video page the video is actually there and it is playable, no capcha, or anything like that
![image](https://user-images.githubusercontent.com/46431157/61625974-07354600-ac4a-11e9-8639-2b36709d3f73.png)

**Comment by thorio** [comment]
To clarify, it's called "experimental" for a reason. See the [explanation page](https://github.com/thorio/KGrabber/wiki/Preferences#enable-experimental-grabbers).
It's the same code used for rapidvideo on kissanime, and I figured it might work on kimcartoon too. I found that it breaks after 5-6 episodes due to captcha, so I put it in as experimental.

I was able to replicate the issue with rapidvideo. They just changed the embed domains, so it's a quick fix. The other servers seem fine (with experimental turned off), so I'm not sure why it's not working for you.
